contracts/VoucherHub.sol

```diff
@@ -177,6 +177,25 @@ contract VoucherHub is
         emit VoucherCreated(voucherAddress, owner, voucherExpiration, voucherType, value);
     }
 
+    /**
+     * Top up a voucher by increasing its balance and extending the validity
+     * period before expiration.
+     * @param voucher The address of the voucher.
+     * @param value The amount of credits to top up.
+     */
+    function topUpVoucher(address voucher, uint256 value) external onlyRole(VOUCHER_MANAGER_ROLE) {
```

Comment:
A small suggestion to make the role name more meaningful
(Can be in a separate PR if not in-scope).
```suggestion
    function topUpVoucher(address voucher, uint256 value) external onlyRole(VOUCHERS_MANAGER_ROLE) {
```

Reply:
See [Add todo to rename roles](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/23/commits/2a068f388ae2127dcd471eba7506fa55fe14c7cd) :)